src/user-utilities.js

```diff
@@ -18,7 +14,10 @@ function createUser(firstName, lastName, email, phoneNumber) {
         email,
         phoneNumber,
         orders: [],
-        getFullName,
+        //formats fullName
+        getFullName: function() {
```

Comment:
formatting should be more in line with other function naming

Reply:
I understand that other functions are not defined inline, but I see this is as a trivial function I'd prefer to leave it, since the behavior is the same and PM wants this deployed today @KevinsCodeStorage